python/ct/client/log_client.py

```diff
@@ -209,18 +210,21 @@ def __str__(self):
     def get_response(self, uri, params=None):
         """Get an HTTP response for a GET request."""
         try:
-            return requests.get(uri, params=params, timeout=self._timeout,
-                                verify=self._ca_bundle)
-        except requests.exceptions.RequestException as e:
+            return RequestHandler._build_requests_style_response(
```

Comment:
That's a correct but a bit awkward way for invoking a static method. I suggest using self._build_requests... to match how check_response_status is called.